cpu/samd21/periph/rtt.c

```diff
@@ -27,6 +27,9 @@
 /* guard file in case no RTT device was specified */
 #if RTT_NUMOF
 
+#ifndef RTT_PRESCALER
```

Comment:
Maybe this can be moved to `periph_cpu.h` with a proper doxygen documentation ? This still allows to use a different value in the board `periph_conf.h` I think.